recipes/locopy/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "locopy" %}
+{% set version = "0.3.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ae2d4fa2e43674dd09f015a2ab948d0e6331989cc617467043f308b6faf8ee19
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python
```

Comment:
Should also be `python >=3.6`

Reply:
Thanks @dopplershift Appreciate it. Should be fixed now.